lld/ELF/Arch/AArch64.cpp

```diff
@@ -366,11 +421,13 @@ static void write32AArch64Addr(uint8_t *l, uint64_t imm) {
   write32le(l, (read32le(l) & ~mask) | immLo | immHi);
 }
 
-static void or32le(uint8_t *p, int32_t v) { write32le(p, read32le(p) | v); }
+static void writeBits32le(uint8_t *p, int32_t v, uint32_t mask) {
```

Comment:
Perhaps `writeMaskedBitsle`?

Just a suggestion, not got a strong opinion, could make the third parameter more obvious at the callsite.